src/tribol/mesh/CouplingScheme.cpp

```diff
@@ -921,9 +937,17 @@ int CouplingScheme::apply( integer cycle, real t, real &dt )
 
    } // end loop over pairs
 
+   // TODO refine how this logging is handled. This just detects an issue with a face-pair geometry
+   // (which has been skipped over for contact eligibility) and reports this warning. Do we want to 
+   // error out, or let a user detect bad contact behavior, but with a contact interaction that still
+   // runs?
```

Comment:
If you ask me, we should report this back to the host code somehow and let them decide how to proceed.  We definitely should not error out.  In an implicit code doing Newton iterations, we'd likely want to try again with a smaller step.

Reply:
Keep these things in mind for a further refinement PR. I agree we shouldn't error out, but giving the host-code some control of how to proceed will be good.